diff --git a/dist/module.mjs b/dist/module.mjs
index 41419bcf40dec371762c672eef49c3b391273654..4fb25cbf3279dcb140fe6e266930d343ee93e251 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -16,6 +16,9 @@ const WorkerTransformPlugin = (opts) => createUnplugin(() => {
       if (!(id in opts.context.reverseMap)) {
         return;
       }
+      if (code.includes("const __worker_exports__")) {
+        return;
+      }
       const s = new MagicString(code);
       const vals = findExports(code);
       for (const val of vals) {
